projects/text_image_video_to_text/README.md

```diff
@@ -0,0 +1,127 @@
+# Qwen2-VL Model Doc for Align-Anything
+
+## Environment setup
+
+Currently, the Transformer in pypi is not updated to support the latest Qwen2-VL model, so we need to install the forked stable version of the repo by running:
+pip install git+https://github.com/huggingface/transformers.git
+
+## Model
+
+We use the `Qwen2-VL-Chat` model for the text-image-video-to-text task. You can download the model from [here](https://huggingface.co/Qwen/Qwen2-VL-7B-Instruct), or directly use the model id `Qwen/Qwen2-VL-7B-Instruct` in the script.
+
+## Training
+
+### Model SFT
+
+Add a script named `sft_text_image_video_to_text.sh` under the `scripts` file like this:
+
+```bash
+# You can replace it with a local model path
+MODEL_NAME_OR_PATH="Qwen/Qwen2-VL-7B-Instruct"
+# You can replace it with a local dataset path
+TRAIN_DATASETS="../../data"
+TRAIN_DATA_FILE="text_image_video_to_text.json"
+# You can replace it with a new path
+OUTPUT_DIR="../outputs/sft_qwen2_vl"
```

Comment:
This is the simplest example we provide. You can refer to their format and replace them with your own dataset.

Reply:
Added in https://github.com/PKU-Alignment/align-anything/pull/68/commits/04a007fefd4b5bd14cc363083646917736eec893